File: AnjaProjekat/Server/UserService/Model/Product.cs
namespace UserService.Model
{
    public class Product :EntityBase
    {
        public string Name { get; set; }
        public double Price { get; set; }
        public int Amount { get; set; }
        public string Description { get; set; }
        public byte[]? Picture { get; set; }
        public virtual User? Seller { get; set; }
        public long SellerId { get; set; }
        public List<OrderProduct> OrderProducts { get; set; }
    }
}

File: Projekat/UI/src/models/Product.js
export class Product {
    constructor(product) {
        this.id = product.id
        this.name = product.name;
        this.description = product.description;
        this.amount = product.amount;
        this.price = product.price;
        this.picture = product.picture
        this.sellerId = product.sellerId
    }
}
export default Product;